pattoo/configuration.py

```diff
@@ -212,6 +214,122 @@ def ip_bind_port(self):
             result = int(intermediate)
         return result
 
+    def jwt_secret_key(self):
+        """Get jwt_secret_key.
+
+        Args:
+            None
+
+        Returns:
+            result: result
+
+        """
+        # Initialize key variables
+        key = PATTOO_API_WEB_NAME
+        sub_key = 'jwt_secret_key'
+
+        # Process configuration
+        result = search(key, sub_key, self._server_yaml_configuration)
+
+        # Ensures that jwt_secret_key is set
+        if (result is None) or result.strip(' ') == '':
+            raise Exception('Plese set JWT SECRET KEY in config file')
```

Comment:
Also use `pattoo_shared.log2die` or `pattoo_shared.log2debug`, that way the message appears in the logs. This won't allow proper troubleshooting.